.gitignore

```diff
@@ -44,3 +44,4 @@ lib*.dylib
 .vs/
 /tags
 /host_tools.mwc*
+*.recipe

```

Comment:
Could this be in local configuration instead of in the repo?

Reply:
https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally